docs/pages/management/operations/backup-restore.mdx

```diff
@@ -137,6 +133,18 @@ $ sudo teleport start --config fresh.yaml --bootstrap state.yaml
 $ tctl --config fresh.yaml get all
 # <your state here>
 ```
+<Notice>
+The `tctl get all` must be run on a Teleport Auth Service host 
+or existing User records will cause a permission error.  This is due
+to User secrets only being available from a a direct `tctl` connection
+on a Teleport Auth Service machine. As a workaround, retrieve resources
+individually with the `--with-secrets` parameter except for users. Local user secrets
+including passwords would need to be reset after creating without secrets.
+
+All trusted cluster certificate resources should be removed for use
+in a bootstrap or you will receive an error.  These are the `kind: cert_authority`
```

Comment:
What does "removed for use in a bootstrap" mean? Is there another way to phrase this?